src/Sidebar/MenuItem.tsx

```diff
@@ -237,8 +233,7 @@ export const MenuItem: React.FC<MenuItemProps> = ({
                       onClick={(event: React.MouseEvent) =>
                         handleClick(event, subMenuItem)
                       }
-                      data-test="submenu-item-label"
-                      data-test-id={subMenuItem.id}
```

Comment:
@krzysztofzuraw   need your expertise how to deal with entries like this, why are we using both of them atm?

Reply:
I propose we merge those two into `data-text-id="submenu-item-label-${subMenuItem.id}"` and then change selectors inside cypress dashboard configuration.